packages/web/integrations/bridges/skip/types.ts

```diff
@@ -0,0 +1,194 @@
+export type Asset = {
+  denom: string;
+  chain_id: string;
+
+  origin_denom: string;
+  origin_chain_id: string;
+  trace: string;
+  is_cw20: boolean;
+
+  symbol?: string;
+  name?: string;
+  logo_uri?: string;
+  decimals?: number;
+  token_contract?: string;
+  description?: string;
+  coingecko_id?: string;
+  recommended_symbol?: string;
+};
+
+export type Chain = {
```

Comment:
Same here, but it would be called 
```suggestion
export type SkipChain = {
```